src/panels/ClusterPropertiesPanel.ts

```diff
@@ -204,17 +211,24 @@ export class ClusterPropertiesDataProvider implements PanelDataProvider<"cluster
             return;
         }
 
-        webview.postGetPropertiesResponse(asClusterInfo(cluster.result));
+        const kubernetesVersion = await getKubernetesVersionInfo(this.client, cluster.result.location);
+        if (failed(kubernetesVersion)) {
+            webview.postErrorNotification(kubernetesVersion.error);
+            return;
+        }
+
+        webview.postGetPropertiesResponse(asClusterInfo(cluster.result, kubernetesVersion.result));
     }
 }
 
-function asClusterInfo(cluster: ManagedCluster): ClusterInfo {
+function asClusterInfo(cluster: ManagedCluster, kubernetesVersionList: KubernetesVersionListResult): ClusterInfo {
     return {
         provisioningState: cluster.provisioningState!,
         fqdn: cluster.fqdn!,
         kubernetesVersion: cluster.kubernetesVersion!,
```

Comment:
I just figured out something: this should be `cluster.currentKubernetesVersion`, not `cluster.kubernetesVersion`.

`kubernetesVersion` may not include the patch version (as is the case for the cluster you were looking at):

```sh
az aks show -n tats-jose-inspektorgadget-demo-2 -g tats-demo | grep -i kubernetesVersion
  "currentKubernetesVersion": "1.27.9",
  "kubernetesVersion": "1.27",
```